eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -255,7 +255,7 @@ public void sendResponse(final ChannelHandlerContext ctx, final DefaultFullHttpR
      * @param httpRequest
      * @return
      */
-    private Map<String, Object> parseHttpRequestBody(final HttpRequest httpRequest) throws IOException {
+    protected Map<String, Object> parseHttpRequestBody(final HttpRequest httpRequest) throws IOException {
         final long bodyDecodeStart = System.currentTimeMillis();
```

Comment:
`parseHttpRequestBody` and `decodeHttpRequestBody` seems to be an util method. Since `parseHttpRequestBody` is used by other classes, will it be better to place them somewhere else like an util class?